orchestration/scripts/check-existing-file.py

```diff
@@ -0,0 +1,26 @@
+import google.auth
+from google.auth.transport.requests import AuthorizedSession
+from requests.exceptions import HTTPError
+import os
+
+credentials, project = google.auth.default(scopes=['openid', 'email', 'profile'])
+
```

Comment:
Maybe add a `print(credentials.client_id)` here so we can see if the load is working?